cmd/immortal/main.go

```diff
@@ -49,11 +48,22 @@ func main() {
 	if len(cfg.Require) > 0 {
 		down := []string{}
 		ctl := &immortal.Controller{}
+		services, _ := ctl.FindServices(immortal.GetSdir())
+		if userServices, err := ctl.FindServices(
+			immortal.GetUserdir(),
```

Comment:
This could be implemented but instead of adding it to the existing services just specify the user `$HOME/.immortal` path